test/time_series_utils.jl

```diff
@@ -182,11 +182,13 @@ function test_file_to_array()
     data, header = PSRI.file_to_array_and_header(
         PSRI.OpenBinary.Reader,
         FILE_PATH,
+        false,
```

Comment:
The examples should not change, I think you should add a new section illustrating the use case of `header = true`.